source/slang/slang-ast-dump.cpp

```diff
@@ -679,33 +682,11 @@ void Context::dumpObjectFull(NodeBase* node)
     }
 }
 
-void Context::dumpObjectFull(Substitutions* subs)
```

Comment:
I like that just putting `Substitutions` into the `NodeBase` hierarchy allows us to remove this code.